### DISCUSSION TITLE
About copliot still can't be used for free after the student certification is passed

### DISCUSSION THREAD

**Discussion by wallbf** [community/community#52709] [GitHub Education]
After my student certification was approved, I still couldn't use copliot, he prompted me that I still need to pay
![image](https://user-images.githubusercontent.com/44938096/231721397-3be1ed84-0083-49de-b42e-6b8d2fa41e6e.png)
![image](https://user-images.githubusercontent.com/44938096/231721516-dbe4a851-85e0-4fa2-bc6c-b50494145ad4.png)
![image](https://user-images.githubusercontent.com/44938096/231721126-d77dc7a4-78c5-4478-b5d9-324868d838c0.png)

**Comment by samus-aran** [comment]
Hi @wallbf 👋 

Confirm that you are a verified student by viewing your [application status](https://education.github.com/discount_requests/pack_application). We sometimes ask students to re-verify themselves.
